MongoDB/containers/0.1.0/mongodb-config/connect.sh

```diff
@@ -28,6 +28,18 @@ function find_master {
 	done
 	return 1
 }
+
+function router_init {
+	for member in $($GIDDYUP ip stringify --delimiter " "); do
```

Comment:
Ah yes! Thanks for pointing this! I don't know why I did this loop, probably a bad monday morning... -_- 
I'll do a new commit when I have time.

I don't know if you've already seen it but I've noticed a problem with this fresh PR of Mongo image: https://github.com/docker-library/mongo/pull/161 
They've added a test which don't work with config server, so they're always recreated in Rancher. 
You should use a ancient tag for the Mongo image, for example `3.0.13` instead of the latest`3.0`.